src/manager.rs

```diff
@@ -83,37 +73,48 @@ impl Manager {
 
     /// Return the open env at `path`, returning `None` if it has not already been opened.
     pub fn get<'p, P>(&self, path: P) -> Result<Option<Arc<RwLock<Rkv>>>, ::std::io::Error>
-    where P: Into<&'p Path> {
+    where
+        P: Into<&'p Path>,
+    {
         let canonical = canonicalize_path(path)?;
         Ok(self.environments.get(&canonical).cloned())
     }
 
     /// Return the open env at `path`, or create it by calling `f`.
     pub fn get_or_create<'p, F, P>(&mut self, path: P, f: F) -> Result<Arc<RwLock<Rkv>>, StoreError>
-    where F: FnOnce(&Path) -> Result<Rkv, StoreError>,
-          P: Into<&'p Path> {
+    where
+        F: FnOnce(&Path) -> Result<Rkv, StoreError>,
+        P: Into<&'p Path>,
+    {
         let canonical = canonicalize_path(path)?;
         Ok(match self.environments.entry(canonical) {
             Entry::Occupied(e) => e.get().clone(),
             Entry::Vacant(e) => {
                 let k = Arc::new(RwLock::new(f(e.key().as_path())?));
                 e.insert(k).clone()
-            }
+            },
         })
     }
 
     /// Return the open env at `path` with capacity `capacity`,
     /// or create it by calling `f`.
-    pub fn get_or_create_with_capacity<'p, F, P>(&mut self, path: P, capacity: c_uint, f: F) -> Result<Arc<RwLock<Rkv>>, StoreError>
-    where F: FnOnce(&Path, c_uint) -> Result<Rkv, StoreError>,
-          P: Into<&'p Path> {
+    pub fn get_or_create_with_capacity<'p, F, P>(
+        &mut self,
+        path: P,
+        capacity: c_uint,
+        f: F,
+    ) -> Result<Arc<RwLock<Rkv>>, StoreError>
+    where
+        F: FnOnce(&Path, c_uint) -> Result<Rkv, StoreError>,
+        P: Into<&'p Path>,
+    {
```

Comment:
It looks like this is the only line that exceeds the new length limit, and it's indeed a really long line, so it's worth wrapping it.